src/frontend/features/Profile/Profile.jsx

```diff
@@ -1,19 +1,28 @@
 import { useSelector } from "react-redux";
+import { useParams } from "react-router-dom";
 import ModalCard from "../../components/ModalCard/ModalCard";
 import ProfileUpdate from "../../components/ProfileUpdate/ProfileUpdate";
```

Comment:
Suggestion: Barrel Exports can be a good idea here to streamline and simplify imports-exports